MQTicketService/Dockerfile

```diff
@@ -10,8 +10,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-FROM docker.io/library/maven:3.6-openjdk-11 AS build
+#
```

Comment:
copyright date needs to be modified to `# © Copyright 2019, 2024 IBM Corporation`